src/SearchBar.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import { PropTypes } from 'react';
+
+class SearchBar extends Component {
+  render() {
+    return (
+      <input
+        className="search-bar"
+        type="text"
+        value={this.props.value}
+        onChange={(event) => this.props.onChange(event)}
```

Comment:
Either check for undefined before calling `this.props.onChange` or declare it as a required PropType.